common/men_at_arms_types/00_maa_types.txt

```diff
@@ -300,14 +424,15 @@ house_guard = {
 	screen = 24
 	
 	counters = {
-		pikemen = 1
+		pikemen = 2
```

Comment:
I see other factors of 2 is reverted, but this one is still here. Is it intended?